src/suggestions/index.jsx

```diff
@@ -88,6 +92,10 @@ export const SuggestionsMixin = {
     const textArea = this.replyInput;
     const textAreaClientRect = textArea.getBoundingClientRect();
 
+    this.suggestionsAbove =
+      suggestionList.offsetHeight > window.innerHeight - textAreaClientRect.top &&
+      suggestionList.offsetHeight < textAreaClientRect.top;
```

Comment:
Is there any way to add a check before this? Just a note that this will trigger a reflow when we measure node dimensions.